lib/resources/PaymentItem.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var IyzipayResource = require('iyzipay/lib/IyzipayResource');
+
+function PaymentItem() {
+    this._config = arguments[0];
+    this._api = {
+        update: {
+            path: '/payment/item',
+            method: 'PUT',
+            requestModel: 'UpdateSubMerchantPaymentItemRequest'
+        }
+    };
+}
+
+PaymentItem.prototype = new IyzipayResource();
+
```

Comment:
the tests related to this function have failed, you have to change IyzipayResource to `var IyzipayResource = require('../IyzipayResource'); `

Reply:
Resolved the issue. That was because of `eslint` formatting. This tells us not to work inside `node_modules` folder :))